.github/workflows/publish_docker.yml

```diff
@@ -0,0 +1,163 @@
+name: Build Production
+on:
+  push:
+    branches:
+      - "main"
+jobs:
+  release:
```

Comment:
Also the release step should be reading the version tag from file right? Since we don't use semantic release in supavisor repo